README.md

```diff
@@ -40,6 +40,8 @@ The traces generation parameters could be simply provided via command line optio
 - `--errorRate=10` - Percentage of erroneous traces (0-100). Default: 0
 - `--rate=50` - Rate at which the spans will be ingested (integer number of spans per second). Default: 100
 - `--traceTypesCount=5` - Number of traces types for auto-generation. Default: 0
+- `--totalTraceCount=16` - Number of traces for auto-generation. If this parameter greater
```

Comment:
From the description is not clear that this parameter can be used in Advanced way also. So if this parameter is set and traceTypesPatterns are defined the generated traces count will be the input value.

Reply:
totalTraceCount exists in the Advanced mode description (line 130).
Do you need more details in this section?

Reply:
Removed "auto" prefix.